Automattic/wp-calypso
# Tailored Flows: Link in Bio intro restyle

## Proposed Changes

Add new link in bio intro design.

<img width="2048" alt="image" src="https://user-images.githubusercontent.com/2653810/184629817-9acdeb80-53e1-432e-8202-bff522083651.png">

<img width="407" alt="image" src="https://user-images.githubusercontent.com/2653810/184629874-e4800849-527c-4fe8-b602-000bca9cc155.png">


## Testing Instructions
- Pull this branch and run yarn start from the root folder
- navigate to `http://calypso.localhost:3000/setup/intro?flow=link-in-bio&siteSlug=YOURSIDE.wordpress.com`
- The designs should match

Related to #66564
Fixes #66564
